controller/WebController.php

```diff
@@ -76,6 +78,13 @@ public function __construct($model)
             $this->honeypot->disable();
         }
         $this->twig->addGlobal('honeypot', $this->honeypot);
+        
+        $direction_test = new Twig_Function('direction_test', function($locale=''){
+            $data = Data::get('layout', $locale);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Punic\Data' in method '__construct'.](https://app.codacy.com/manual/NatLibFi/Skosmos/pullRequest?prid=4951454)